hbw/app/assets/stylesheets/hbw/brand_variables.sass

```diff
@@ -65,3 +65,184 @@
   --base-select-dropdown-indicator-color-hover: #333
   --base-select-dropdown-indicator-color-focus: #666
   --base-select-dropdown-indicator-width: 36px
+
+  //Fonts 
+  --base-hbw-default-font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif  
+
+  //Panel title
+  --base-hbw-panel-title-font-family: var(--base-hbw-default-font-family)
+  --base-hbw-panel-title-font-size: 17px
+  --base-hbw-panel-title-font-weight: normal
+  --base-hbw-panel-title-font-style: normal
+  
+  // Process name
+  --base-hbw-process-name-font-family: var(--base-hbw-default-font-family)
+  --base-hbw-process-name-font-size: 1.5em
+  --base-hbw-process-name-font-weight: normal
+  --base-hbw-process-name-font-style: normal
+
+  // BP tabs
+  --base-hbw-nav-tabs-font-family: var(--base-hbw-default-font-family)
+  --base-hbw-nav-tabs-font-size: var(--base-font-size)
+  --base-hbw-nav-tabs-font-weight: 400
```

Comment:
It's normal.

Reply:
![Screenshot 2021-04-01 at 11 28 02](https://user-images.githubusercontent.com/36504360/113265939-5bc50b80-92dd-11eb-815d-b262fd041a86.png)